### PR TITLE
Hunter Buff.

### PR DESCRIPTION
**Pull request by vilereaver41**
## About The Pull Request

makes hunter more viable as a t2 caste. its skillset is based on ambushing/flanking rather than frontline combat. For a cost of course.

## Why It's Good For The Game

hunter is underperforming really badly right now, this makes it more rewarding, hunter in itself is not a frontline caste it is meant for taking out solo marines, weak marines, or miners. 

 

## Changelog
:cl:
add: new descriptions to hunter examine comments.
balance: changed hunter damage values to increase fairly per upgrade. Young & Mature 21 to 22.  Elder stays at 24. Ancient 26.
balance: increased hunter speed per tier.
balance: increase sneak attack stagger values as well as slowdown values. values are increased by 1 respectively.
balance: increase pounce cost to 80 plasma.
balance: decreased pounce cooldown timer per tier.
balance: increase plasma drain while the hunter is moving as they are Invisible.
balance: decreased attack delay value.
balance: increase hunter upgrade threshold values per tier.
balance: adjusted health values per tier.
balance: increase hunter armor values per tier.
balance: adjusted plasma pool values per tier. /:cl:

### CODE REVIEW

**Comment by Pariah919** [comment]
Our melee damage is tiered in two per castes.

Also this doesn't fix hunter's core problems which is the fact that has nothing to bring to the team at all.

**Comment by David-Stormwell** [comment]
I'd suggest giving a brief explanation of what numbers you changed on the description as well.

File: code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm [hunk 1]
```diff
@@ -98,7 +98,6 @@
 	stealth = TRUE
 	RegisterSignal(owner, COMSIG_MOVABLE_MOVED, .proc/handle_stealth)
 	handle_stealth()
-	ADD_TRAIT(owner, TRAIT_TURRET_HIDDEN, STEALTH_TRAIT)
```
Comment (QualityVan) [inline code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm:101]: ?

Comment (vilereaver41) [reply code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm:101]: no idea why thats there, it showed up after i pulled it from main tgmc. gona remove it.

Comment (vilereaver41) [inline code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm:101]: This isnt even there lol, wasnt staged, repo was copied from main tgmc.

File: code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm [hunk 2]
```diff
@@ -110,7 +109,6 @@
 	UnregisterSignal(owner, COMSIG_MOVABLE_MOVED) //This should be handled on the ability datum or a component.
 	stealth = FALSE
 	can_sneak_attack = FALSE
-	REMOVE_TRAIT(owner, TRAIT_TURRET_HIDDEN, STEALTH_TRAIT)
```
Comment (vilereaver41) [inline code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm:113]: same as above comment

**Comment by vilereaver41** [comment]
> Our melee damage is tiered in two per castes.
> 
> Also this doesn't fix hunter's core problems which is the fact that has nothing to bring to the team at all.

The first point, well it doesn't hurt to give it a try. the second point. while wraith is a saboteur, hunter is meant to give marine backlines and people going back and forth or scouting ahead a hell of a time so that marines are not able to consolidate obtained ground so good.

**Comment by Naaanii** [comment]
Buffing young stats doesn't do much since young T2's don't really exist any more. This looks like a more cumbersome revert of the previous nerfs and it'd just end up going back to square 1 and doesn't really change any of the problems it had before.

**Comment by vilereaver41** [comment]
> Buffing young stats doesn't do much since young T2's don't really exist any more. This looks like a more cumbersome revert of the previous nerfs and it'd just end up going back to square 1 and doesn't really change any of the problems it had before.

The previous hunter was too strong, the current hunter is too weak. I like to think that my current tm will have balanced in between, in the end, if something is not trialed via merge and things are balanced based on "this looks like this or that" then you wont be able to fix a caste, have you considered all the additions and changes for marines  since hunter was nerfed to begin with?

**Comment by TiviPlus** [comment]
What others said, this doesnt adress the core issues with hunter

**Comment by TiviPlus** [comment]
As a side note this is also very unatomized, split your prs up more in future

**Pull request closed without merging**